Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/08-DNS-Records/dns-records.tf

```diff
@@ -0,0 +1,8 @@
+resource "azurerm_dns_a_record" "app_record" {
```

Comment:
Please indicate that this resource is only used for the B/G scenario

Reply:
fixed